mipengine/controller/api/DTOs/AlgorithmSpecificationsDTOs.py

```diff
@@ -0,0 +1,150 @@
+from dataclasses import dataclass
+from typing import List, Optional, Dict
+
+from dataclasses_json import dataclass_json
+
+from mipengine.controller.common.algorithms_specifications import CROSSVALIDATION_ALGORITHM_NAME, \
+    GenericParameterSpecification, AlgorithmSpecifications, AlgorithmsSpecifications
+from mipengine.controller.common.utils import Singleton
+
+INPUTDATA_PATHOLOGY_PARAMETER_NAME = "pathology"
+INPUTDATA_DATASET_PARAMETER_NAME = "dataset"
+INPUTDATA_FILTERS_PARAMETER_NAME = "filter"
+INPUTDATA_X_PARAMETER_NAME = "x"
+INPUTDATA_Y_PARAMETER_NAME = "y"
+
+
+@dataclass_json
+@dataclass
+class InputDataSpecificationDTO:
+    """
+    InputDataParameterDTO is different from the InputDataParameter
+    on the stattypes field.
+    It is optional on the DTOs, due to the dataset and pathology parameters.
+    """
+    label: str
+    desc: str
+    types: List[str]
+    notblank: bool
+    multiple: bool
+    stattypes: Optional[List[str]] = None
+    enumslen: Optional[int] = None
+
+
+def get_pathology_parameter():
+    return InputDataSpecificationDTO(
+        "Pathology of the data.",
+        "The pathology that the algorithm will run on.",
+        ["text"],
+        True,
+        False,
+        None,
+        None,
+    )
```

Comment:
When you have many arguments to function calls it's a good idea to pass them in keyword format. E.g. here I can't tell what `True, False, None, None` are referring to.

Reply:
Done.